src/io/flutter/run/LaunchState.java

```diff
@@ -119,7 +124,7 @@ private RunContentDescriptor launch(@NotNull ExecutionEnvironment env) throws Ex
       }
     }
 
-    final FlutterLaunchMode launchMode = FlutterLaunchMode.getMode(env);
+    final FlutterLaunchMode launchMode = FlutterLaunchMode.getFromEnvironment(env);
```

Comment:
For parallelism, maybe this could be `fromEnv(..)` like `RunMode.fromEnv(..)`?

Reply:
👍

Reply:
will do